test/conformance/walk.go

```diff
@@ -254,7 +486,7 @@ func (v *visitor) firstArg(n *ast.CallExpr) string {
 	if ident, ok := n.Args[0].(*ast.Ident); ok {
 		return ident.String()
 	}
-	return "*"
+	return ""
```

Comment:
Can we make sure this is consistent with the comment describing what the function returns?

Reply:
Thanks for pointing. The codebase is updated as requested. Though I did not fully understand the benefit of replacing %s/%d with "*" from the first arg of all "Describe|KubeDescribe|SIGDescribe|Context" if arg[0] is non-string literal.